### ISSUE TITLE
JAVA - How to get channel members

### ISSUE THREAD

**Issue by alon-sys** [tdlib/td#2157]
How to get channel members, which api is it?

**Comment by levlam** [comment]
See https://github.com/tdlib/td/issues/1478.

**Comment by alon-sys** [comment]
> 

Error {
  code = 400
  message = "CHAT_ADMIN_REQUIRED"
}

**Comment by levlam** [comment]
Of course, you can't get channel members without administrator rights.

**Comment by alon-sys** [comment]
> Of course, you can't get channel members without administrator rights.

THX
